maintain/going-production-with-k8s.md

```diff
@@ -0,0 +1,429 @@
+title: "Operations Guide to deploy Portworx in Production in Kubernetes Clusters"
+keywords: kuberntes, operations guide, run book, disaster recovery, disaster proof, site failure, node failure, power failure
+sidebar: home_sidebar
+meta-description: "Portworx Operations Guide for Kubernetes Deployments"
+---
+
+* TOC
+{:toc}
+
+
+## DAY 1 Operations
+
+### Initial Software Setup for Production
+ 
+  * Follow the instructions in the [k8s install](https://docs.portworx.com/scheduler/kubernetes/install.html) 
+    page in the docs. 
+  * Ensure all nodes in the cluster have NTP running and the times are synchronized across all the nodes that will 
+    form the Portworx cluster
+  * All nodes in the cluster should have achieved quorum and `pxctl status` should display the cluster as `operational`
+  * etcd -  Setup etcd as a 3-node etcd cluster *outside* the  container orchestrator to ensure maximum stability. 
+    Refer to the following [page](https://docs.portworx.com/maintain/etcd.html) on how to install etcd and also 
+    configure it for maximum stability.
+
+### Configuring the Server or the Compute Infrastructure
+
+* Check and ensure a *minimum* 2 cores and 4GB of RAM are allocated for Portworx. 
+* The base operating system of the server supports linux kernel 3.10+ . Newer 4.x linux kernels have many performance and stability related fixes and is recommended.
+
+```
+[centos@ip-172-31-51-89 ~]$ uname -r
+3.10.0-327.22.2.el7.x86_64
+```
+
+### Configuring the Networking Infrastructure
+
+* Make sure the following ports are open in all the servers. 9001-9014
+
+* Configure separate networks for Data and Management networks to isolate the traffic
+
+  * Data network is specified giving the '-d' switch and Management networks with the '-m' switch. Refer to [scheduler guides](https://docs.portworx.com/#install-with-a-container-orchestrator) for specifics to enable it in your scheduler.
+  
+  * With multiple NICs, create a bonded ethernet port for data interface for improved availability and performance.
+  
+### Configuring and Provisioning Underlying Storage
+
+####  Selecting drives for an installation
+
+* Storage can be provided to Portworx explicitly by passing in a list of block devices. `lsblk -a` will display a list of devices on the system. This is accomplished by the '-s' flag as a runtime parameter. It can also be provided implicity by passing in the '-a' flag. In this mode, Portworx will pick up all the available drives that are not in use. When combined with '-f', Portworx will pick up drives even if they have a filesystem on them (mounted drives are still excluded).  Note that not all nodes need to contribute storage; a node can operate in the storageless mode with the '-z' switch. Refer to [scheduler guides](https://docs.portworx.com/#install-with-a-container-orchestrator) for specifics for your scheduler.
+
+* HW RAID - If there are a large number of drives in a server and drive failure tolerance is required per server, 
```

Comment:
Do you need to specify HW? Software raid would work as well, plus we do provide a mirroring option for dealing with drive failures.

Reply:
No. Large number of drives and drive tolerance per server require HW raid